src/builtins/register.rs

```diff
@@ -184,10 +187,12 @@ pub async fn register_application_commands_buttons<U, E>(
         .await;
 
     reply.edit(ctx, |b| b.components(|b| b)).await?; // remove buttons after button press
+    // NOTE: Can this be done in one .edit?
+    reply.edit(ctx, |b| b.content("Processing... Please wait.")).await?; // Edit message to processing after button press 
```

Comment:
This should be possible to do in one edit, if you just chain the builder function calls after another `.components(...).content(...)`